packages/zowe-explorer/package.json

```diff
@@ -823,7 +823,7 @@
       },
       {
         "command": "zowe.placeholderCommand",
-        "title": "%zowe.placeholderCommand%",
+        "title": "",
         "enablement": "false"
       }
```

Comment:
Dumb questions: 😋 
- What's this?
- Can we remove it?

Reply:
We use this to show tree nodes as disabled. VS Code will "gray out" a tree node that has a command assigned with "enablement = false". Thanks @traeok for suggesting this trick 😋